synapsis/src/main/java/com/soen/synapsis/appuser/AppUserService.java

```diff
@@ -36,4 +36,14 @@ public String signUpUser(AppUser appUser) {
 
         return "pages/home";
     }
+
+    public void markCandidateToRecruiter(AppUser appUser) {
+
+        if(appUser.getRole() != Role.CANDIDATE) {
+            throw new IllegalStateException("The user must be a candidate to be marked as a recruiter.");
+        }
+        appUser.setRole(Role.RECRUITER);
```

Comment:
We seem to be setting the candidate to a recruiter correctly, but we're not marking which company they're a recruiter for. Can you add a column to the AppUser table that contains a foreign key to the company id that is adding them?

Reply:
Does this mean I need to create a new entity Company?

Reply:
This should work now.